src/main/kotlin/uk/gov/gdx/datashare/queue/AmazonSnsFactory.kt

```diff
@@ -0,0 +1,38 @@
+package uk.gov.gdx.datashare.queue
+
+import com.amazonaws.auth.AWSStaticCredentialsProvider
+import com.amazonaws.auth.BasicAWSCredentials
+import com.amazonaws.client.builder.AwsClientBuilder
+import com.amazonaws.services.sns.AmazonSNS
+import com.amazonaws.services.sns.AmazonSNSClientBuilder
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+class AmazonSnsFactory {
+  companion object {
+    val log: Logger = LoggerFactory.getLogger(this::class.java)
+  }
+
+  fun awsSnsClient(topicId: String, accessKeyId: String, secretAccessKey: String, region: String): AmazonSNS =
```

Comment:
Why does this take a topicId? Is it just for logging?

Reply:
Yeah, not sure how helpful the logging is right now, maybe we should do a cull of it at some point